app/components/Form/Field.js

```diff
@@ -8,7 +8,11 @@ import Tooltip from 'app/components/Tooltip';
 import styles from './Field.css';
 
 const FieldError = ({ error }: { error?: string }) =>
-  error ? <div className={styles.fieldError}>{error}</div> : null;
+  error ? (
+    <div className={styles.fieldError}>
+      {typeof error === 'object' ? JSON.stringify(error) : error}
```

Comment:
It looked like the error this morning was caused by this line trying to render an object. Not 110% sure tho. This will atleast help with not allowing to render objects.

Reply:
Smart